docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:10.13.0-alpine as production-build
+COPY .  .
+
+# Install yarn
+RUN npm install -g yarn
+
+# Install only production dependencies
+RUN yarn install
```

Comment:
isn't this the command for only production dependencies? 
`yarn install --production`

Reply:
Yes,  actually, we don't have *node_modules* in the final image. We are only installing packages required to build the Artifact.
Using the `dev-dependencies` on this step will only make the process take longer.

Reply:
so, just a follow-up on this, don't we want to change the `yarn install` so that it doesn't install the `dev-dependencies`?